src/core/api/project.go

```diff
@@ -532,6 +532,12 @@ func validateProjectReq(req *models.ProjectRequest) error {
 		return fmt.Errorf("project name is not in lower case or contains illegal characters")
 	}
 
+	validProjectName = regexp.MustCompile(`api(?:[._-][a-z0-9]+)*`)
+	legal = validProjectName.MatchString(pn)
+	if legal {
+		return fmt.Errorf("project name is not started with 'api'")
```

Comment:
s/is not started with/may not start with/

Reply:
https://github.com/goharbor/harbor/blob/d9b0ef4618ceb493de046dd615e1212d17e4891e/src/core/api/project.go#L537-L539

Shouldn't this be ```if !legal```?